backend/src/org/commcare/util/CommCarePlatform.java

```diff
@@ -50,7 +50,20 @@ public CommCarePlatform(int majorVersion, int minorVersion) {
         this.minorVersion = minorVersion;
     }
 
-    public void init(String profileReference, ResourceTable global, boolean forceInstall) throws UnfullfilledRequirementsException, UnresolvedResourceException {
+    /**
+     * Installs resources described by profile reference into the provided
+     * resource table. If the resource table is ready or already has a profile,
+     * don't do anything.
+     *
+     * @param profileReference URL to profile file
+     * @param global           Add profile ref to this table and install its
+     *                         resources
+     * @param forceInstall     Should installation be performed regardless of
+     *                         version numbers?
```

Comment:
Is this the correct comment for 'forceInstall'? It is passed along all the way down to its cryptic use in ProfileParser (https://github.com/dimagi/commcare/blob/master/backend/src/org/commcare/xml/ProfileParser.java#L64)